test/integration/test_ibm_job.py

```diff
@@ -263,8 +263,8 @@ def test_retrieve_jobs_status(self):
 
         for job in backend_jobs:
             self.assertTrue(
-                job.status() is JobStatus.DONE,
-                "Job {} has status {} when it should be DONE".format(
+                job.status() is JobStatus.DONE or job.status() is JobStatus.CANCELLED,
+                "Job {} has status {} when it should be DONE or CANCELLED".format(
```

Comment:
@kt474 Thanks for looking into this! We also need to add `JobStatus.ERROR` to this since that is also one of the final states. We can use the JOB_FINAL_STATES tuple and check if the job.status() is in it or not.